spec/models/green_lanes/category_assessment_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+RSpec.describe GreenLanes::CategoryAssessment do
+  subject(:category_assessment) { build(:category_assessment) }
+
+  describe '.relationships' do
+    let(:expected_relationships) do
+      %i[
+        geographical_area
+        excluded_geographical_areas
+      ]
+    end
+
+    it { expect(described_class.relationships).to match_array(expected_relationships) }
```

Comment:
Rather than one 'test all the relationships' spec - this could be broken out into separate tests which would individually fail highlighting the origin of any future test failure, eg

```ruby
describe '.relationships' do
  subject { category_assessement.relationship }

  it { is_expected.to include :geographical_area }
  it { is_expected.to include :excluded_geographical_areas }
end
```